Auth

yannmtrn opened the pull request:
Adding signin page, signup page and traitment of these

FBanitz committed:
Merge pull request #3 from CESI-CDA-2022-23-Groupe-4/users-table

Added users table

yannmtrn committed:
Merge branch 'auth' of https://github.com/CESI-CDA-2022-23-Groupe-4/pandanim into auth

FBanitz committed:
Merge branch 'dev' into auth

yannmtrn committed:
Merge branch 'auth' of https://github.com/CESI-CDA-2022-23-Groupe-4/pandanim into auth

FBanitz committed:
Create index.twig

FBanitz committed:
Merge branch 'auth' of https://github.com/CESI-CDA-2022-23-Groupe-4/pandanim into auth

yannmtrn committed:
Merge branch 'auth' of https://github.com/CESI-CDA-2022-23-Groupe-4/pandanim into auth

yannmtrn committed:
Add register files

yannmtrn committed:
Add user end

yannmtrn committed:
Add Register.php, signin.twig and signup.twig

- Add page signin
- Add page signup
- Add Register.php and traitment signin and signup

yannmtrn committed:
Add env file

FBanitz reviewed (changes requested):
Do not delete env file

FBanitz committed:
fixed kint error

yannmtrn committed:
Merge branch 'dev' into auth

yannmtrn committed:
correction and improvement

- Auth.php renamed from Register.php
- Adding username, firstname and lastname on pandanim.sql and dataDictionnary.md
- Adding init session
- Other small fixed

yannmtrn committed:
Merge branch 'auth' of https://github.com/CESI-CDA-2022-23-Groupe-4/pandanim into auth
